app/assets/javascripts/discourse/app/controllers/user-invited-show.js

```diff
@@ -43,32 +44,26 @@ export default Controller.extend({
   inviteRedeemed: equal("filter", "redeemed"),
   invitePending: equal("filter", "pending"),
 
-  @discourseComputed("filter")
-  inviteLinks(filter) {
-    return filter === "links" && this.currentUser.staff;
-  },
-
   @discourseComputed("filter")
   showBulkActionButtons(filter) {
     return (
       filter === "pending" &&
-      this.model.invites.length > 4 &&
+      this.model.invites.length > 1 &&
       this.currentUser.staff
     );
   },
 
   canInviteToForum: reads("currentUser.can_invite_to_forum"),
   canBulkInvite: reads("currentUser.admin"),
-  canSendInviteLink: reads("currentUser.staff"),
 
-  @discourseComputed("totalInvites", "inviteLinks")
-  showSearch(totalInvites, inviteLinks) {
-    return totalInvites >= 10 && !inviteLinks;
+  @discourseComputed("invitesCount.total")
+  showSearch(invitesCountTotal) {
+    return invitesCountTotal > 0;
   },
 
   @discourseComputed("invitesCount.total", "invitesCount.pending")
   pendingLabel(invitesCountTotal, invitesCountPending) {
-    if (invitesCountTotal > 50) {
+    if (invitesCountTotal > 0) {
```

Comment:
Should this be `showSearch` instead?

Reply:
No, I do not think so. The controls are in no way related and it is just a coincidence that we show the search bar and the count when the `count > 0`.

I could create a new property `hasRecords` and use that instead in `showSearch` too, but I do not think it is worth it given how simple this predicate is.